src/main/java/core/basesyntax/operations/SupplyOperation.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class SupplyOperation implements OperationHandler {
+    private final Storage storage;

```

Comment:
The `storage` field should be private and not exposed directly. Instead of directly accessing the `storage` field from the `SupplyOperation` class, consider using dependency injection to pass the necessary `Storage` object to the `SupplyOperation` class.